app/views/ckey2bibframes/new.html.erb

```diff
@@ -0,0 +1,36 @@
+<h1>Ckey to Bibframe2 Conversion</h1>
```

Comment:
Does _everybody_ know what a `Ckey` is?  Wondering whether this form might be more explicit about what that is exactly?  Maybe explain where to find valid values for this form input?  Does the form input require a form-validation to check that a Ckey exists?